src/components/pages/WordCloud/CropCloud.js

```diff
@@ -22,11 +19,11 @@ export default function CropCloud() {
   //             setCloudImg(res.data)
   //         })
   //         .catch((err) =>{
-  //             console.dir(err)
```

Comment:
Tip here since I saw it on the Loom video, when you highlight what you want to comment out and hit "command + /" it will uncomment for you! Also works when commenting.

Reply:
> 

I am still working on it, the task came from a Trello card that noted the bug on the disappearing crop cloud.